### ISSUE TITLE
ContractInspector 'click here' link broken

### ISSUE THREAD

**Issue by algotrader-au** [tradewright/ibapi-tools#2]
The Click Here to install link in the ContractInspector Read Me file appears broken - or at least for me it just goes to a raw html page.

**Comment by rlktradewright** [comment]
Yes, some Windows users experience this, and some don't. It started happening a couple of years back. If I remember correctly it started when Edge was introduced. It still worked with Internet Explorer, but that's not an option now. 

I reported it to Microsoft at the time, but no-one was able to throw any light on it. There are various settings, which are supposed to govern the handling of the .application file, but they don't seem to work correctly. Quite a number of other users chimed in with the same complaint (for different Click-Once apps), so it's not specific to the Contract Inspector.

The workaround is to right-click in the page of xml, select 'Save as', and save it somewhere suitable (Downloads will do fine).

Then go to the folder you saved it in with File Explorer and double-click on the .application file. It should now run the tool that handles these files and proceed with the installation.

I must add something about this to the description. I've never bothered so far, because no-one else has reported it as far as I recall, so I assumed that it must be something unusual about my setup here.

**Comment by algotrader-au** [comment]
Great thanks - the workround worked fine.  Issue was experienced on Chrome, running Windows 11.
